### PR TITLE
move cpu load out of vehicle_status

### PR DESCRIPTION
**Pull request by jschall**
This comes from an initiative to make uORB messages more atomic and create a commander with minimal functionality.

### CODE REVIEW

**Comment by kd0aij** [comment]
doesn't this burn a lot of extra RAM by creating a process just to calculate and publish cpu load?

File: src/modules/sdlog2/sdlog2.c [hunk 1]
```diff
@@ -1117,6 +1118,7 @@ int sdlog2_thread_main(int argc, char *argv[])
 		struct ekf2_innovations_s innovations;
 		struct vehicle_land_detected_s land_detected;
 		struct commander_state_s commander_state;
+        struct cpuload_s cpuload;
```
Comment (julianoes) [inline src/modules/sdlog2/sdlog2.c]: wrong indentation, I'd say run fix_code_style.sh

File: src/platforms/qurt/px4_layer/commands_hil.c [hunk 2]
```diff
@@ -49,6 +49,7 @@ const char *get_commands()
 		"param set CAL_MAG0_ID 196608\n"
 //		"rgbled start\n"
 //		"tone_alarm start\n"
+		"load_mon start\n"
```
Comment (julianoes) [inline src/platforms/qurt/px4_layer/commands_hil.c]: You also need to add to the startup file for on the Snapdragon DSP in https://github.com/PX4/Firmware/blob/master/posix-configs/eagle/flight/px4-flight.config.

**Comment by julianoes** [comment]
@jschall Nice, reviewed. I made a couple of small comments but overall it looks good and correct.
Also, I'm a bit confused about all the whitespace changes in commander.cpp. Could it be that you haven't rebased on the latest or did I not correctly fix the code style?

**Comment by LorenzMeier** [comment]
@kd0aij Yes, this should be in a work queue. I already discussed this with @jschall.

**Comment by LorenzMeier** [comment]
@julianoes Is this good to go? @jschall This can go in once in a work queue.

**Comment by julianoes** [comment]
I'll have to rebase it once again. Will do tomorrow.

**Comment by LorenzMeier** [comment]
@julianoes can you rebase this and we then pull it to master?

**Comment by julianoes** [comment]
@LorenzMeier I'm planning on getting this in next week.

**Comment by LorenzMeier** [comment]
Yes, happy to support that.

**Comment by julianoes** [comment]
@jschall I'm taking this one over, thanks for your work.
I'm closing this PR in favor of #4396.

**Pull request closed without merging**